locales-src/newtab.ftl

```diff
@@ -148,6 +148,15 @@ newtab-empty-section-highlights = Start browsing, and we’ll show some of the g
 #  $provider (String): Name of the content provider for this section, e.g "Pocket".
 newtab-empty-section-topstories = You’ve caught up. Check back later for more top stories from { $provider }. Can’t wait? Select a popular topic to find more great stories from around the web.
 
+
+## Empty Section (Content Discovery Experience). These show when there are no more pocket stories or when some stories fail to load.
+
+newtab-discovery-empty-section-topstories-header = You are caught up!
+newtab-discovery-empty-section-topstories-content = Check back later for more stories.
+newtab-discovery-empty-section-topstories-try-again-button = Try Again
+newtab-discovery-empty-section-topstories-loading = Loading...
+newtab-discovery-empty-section-topstories-unfinished = Oops! We almost loaded this section, but not quite.
```

Comment:
When is this one displayed?

Reply:
We're unsure how to mention that we have two experiences for a section (very similar and minor differences).

We put a group header here called "Empty Section (Content Discovery Experience)" and there's a group just above this called "Empty Section States". 

I can copy the other section over similarly
## Empty Section States (Content Discovery Experience): These show when there are no more items in a section. Ex. When there are no more Pocket story recommendations, in the space where there would have been stories, this is shown instead.

Reply:
That doesn't really answer my question. Is this displayed when there's an error? Is this displayed together with which strings? Why "unfinished"?

Reply:
This landed as part of #5085 to show a message when we have a layout for a section but took too long / timed out to get the articles.
![Screen Shot 2019-07-13 at 10 54 53 PM](https://user-images.githubusercontent.com/438537/61179895-6a2b3b00-a5c1-11e9-88a8-0aa6f27c448b.png)

Reply:
Then just need a comment to explain that, no need to add it to the section comment.

Reply:
Ah I misunderstood! Sounds good - I'll change from "unfinished" to "timed-out" and I'll add a comment.